### PR TITLE
Cartridge role setup docs

### LINKED ISSUE #163: Cartridge role setup in docs
Now it's not clear how to setup cartridge role
I propose to:

- add picture of cartridge section with config or more detailed explanation of cartridge configuration for metrics, without a link to cartridge configuration in docs
- add note about permanent role in 0.6.0 and steps to enable the role in 0.3.0-0.5.0

### PR DESCRIPTION
**Pull request by ZolotarevIgor-mail-ru**
I've add:
- picture of cartridge section with config or more detailed explanation of cartridge configuration for metrics
- note about permanent role in 0.6.0 and steps to enable the role in 0.3.0-0.5.0
- note about alias label

Close #163

### CODE REVIEW

File: doc/monitoring/getting_started.rst [hunk 1]
```diff
@@ -101,8 +102,18 @@ via configuration.
...
+   And then enable it in interface:
+
+   .. image:: images/role-enable.png
+      :align: center
+
+   Since version **0.6.0** metrics role is permanent and enabled on instances by default.
+
 #. After role initialization, default metrics will be enabled and the global
-   label 'alias' will be set. If you need to use the functionality of any
+   label ``'alias'`` will be set. **Note** that ``'alias'`` label is set by
```
Comment (DifferentialOrange) [inline doc/monitoring/getting_started.rst:113]:
```suggestion
   label ``'alias'`` will be set. **Note** that ``'alias'`` label value is set by
```

File: doc/monitoring/getting_started.rst [hunk 2]
```diff
@@ -77,7 +77,7 @@ via configuration.
 **Usage**
 
 #. Add the ``metrics`` package to dependencies in the ``.rockspec`` file.
```
Comment (DifferentialOrange) [inline doc/monitoring/getting_started.rst:79]:
```suggestion
#. Add ``metrics`` package to dependencies in the ``.rockspec`` file.
```

**Review by DifferentialOrange — APPROVED** [review]

File: doc/monitoring/getting_started.rst [hunk 3]
```diff
@@ -87,7 +87,8 @@ via configuration.
            ...
        }
 
-#. Add ``cartridge.roles.metrics`` to the roles list in ``cartridge.cfg``
+#. If you are using versions **0.3.0-0.5.0** then add ``cartridge.roles.metrics``
```
Comment (DifferentialOrange) [inline doc/monitoring/getting_started.rst:90]:
```suggestion
#. If you are using versions **0.3.0**-**0.5.0** then add ``cartridge.roles.metrics``
```

**Review by vasiliy-t — APPROVED** [review]

File: doc/monitoring/getting_started.rst [hunk 4]
```diff
@@ -101,8 +102,18 @@ via configuration.
            },
        })
 
+   And then enable it in interface:
+
+   .. image:: images/role-enable.png
```
Comment (ZolotarevIgor-mail-ru) [inline doc/monitoring/getting_started.rst:107]: Выделить на скриншоте нужную область

File: doc/monitoring/getting_started.rst [hunk 5]
```diff
@@ -101,8 +102,18 @@ via configuration.
...
+      :align: center
+
+   Since version **0.6.0** metrics role is permanent and enabled on instances by default.
+
 #. After role initialization, default metrics will be enabled and the global
-   label 'alias' will be set. If you need to use the functionality of any
+   label ``'alias'`` will be set. **Note** that ``'alias'`` label value is set by
+   configuration option ``alias``.
```
Comment (ZolotarevIgor-mail-ru) [inline doc/monitoring/getting_started.rst:114]: уточнить

**Review by opomuc — APPROVED** [review]

**Pull request merged**